Updates/Instances/543_hellfire_ramparts.sql

```diff
@@ -163,81 +218,225 @@ REPLACE INTO `creature_template_addon` (`entry`, `mount`, `bytes1`, `b2_0_sheath
+(@SGGUID+14, @CGUID+78, 1), -- Shattered Hand Warhound
+(@SGGUID+14, @CGUID+79, 2), -- Shattered Hand Warhound
+(@SGGUID+15, @CGUID+84, 0), -- Boss Watchkeeper Gargolmar
+(@SGGUID+15, @CGUID+87, 1), -- Hellfire Watcher
+(@SGGUID+15, @CGUID+88, 2), -- Hellfire Watcher
+(@SGGUID+16, @CGUID+107, 0), -- Bonechewer Destroyer
+(@SGGUID+16, @CGUID+108, 1), -- Bonechewer Destroyer
+(@SGGUID+17, @CGUID+56, 0), -- Bonechewer Destroyer
+(@SGGUID+17, @CGUID+36, 1), -- Bleeding Hollow Darkcaster
+(@SGGUID+17, @CGUID+37, 2), -- Bleeding Hollow Darkcaster
+(@SGGUID+18, @CGUID+57, 0), -- Bonechewer Destroyer 
+(@SGGUID+18, @CGUID+46, 1), -- Bleeding Hollow Archer
+(@SGGUID+18, @CGUID+75, 2), -- Shattered Hand Warhound
+(@SGGUID+18, @CGUID+74, 3), -- Shattered Hand Warhound
+(@SGGUID+18, @CGUID+47, 4), -- Bleeding Hollow Archer
+(@SGGUID+19, @CGUID+14, 0), -- Bonechewer Hungerer
+(@SGGUID+19, @CGUID+15, 1), -- Bonechewer Hungerer
+(@SGGUID+20, @CGUID+83, 0), -- Bonechewer Ripper
+(@SGGUID+20, @CGUID+95, 1), -- Bleeding Hollow Scryer
+(@SGGUID+20, @CGUID+96, 2), -- Bleeding Hollow Scryer
+(@SGGUID+20, @CGUID+97, 3), -- Bleeding Hollow Scryer
+(@SGGUID+20, @CGUID+97, 4), -- Bleeding Hollow Scryer
+(@SGGUID+21, @CGUID+58, 0), -- Bonechewer Destroyer
+(@SGGUID+21, @CGUID+59, 1), -- Bonechewer Destroyer
+(@SGGUID+22, @CGUID+50, 0), -- Bonechewer Destroyer
+(@SGGUID+22, @CGUID+76, 1), -- Shattered Hand Warhound
+(@SGGUID+22, @CGUID+1, 2), -- Bonechewer Hungerer
+(@SGGUID+22, @CGUID+32, 3), -- Bleeding Hollow Scryer
+(@SGGUID+22, @CGUID+63, 4), -- Shattered Hand Warhound
+(@SGGUID+23, @CGUID+60, 0), -- Bonechewer Destroyer
+(@SGGUID+23, @CGUID+42, 1), -- Bleeding Hollow Archer
+(@SGGUID+23, @CGUID+43, 2), -- Bleeding Hollow Archer
+(@SGGUID+24, @CGUID+61, 0), -- Bonechewer Destroyer
+(@SGGUID+24, @CGUID+41, 1), -- Bleeding Hollow Archer
+(@SGGUID+24, @CGUID+45, 2), -- Bleeding Hollow Archer
+(@SGGUID+25, @CGUID+19, 0), -- master Bonechewer Ravener
+(@SGGUID+25, @CGUID+62, 1), -- Shattered Hand Warhound
+(@SGGUID+25, @CGUID+67, 2), -- Shattered Hand Warhound
+(@SGGUID+26, @CGUID+17, 0), -- Bonechewer Hungerer
+(@SGGUID+26, @CGUID+28, 1), -- Bleeding Hollow Darkcaster
+(@SGGUID+26, @CGUID+29, 2), -- Bleeding Hollow Darkcaster
+(@SGGUID+26, @CGUID+66, 3), -- Shattered Hand Warhound
+(@SGGUID+26, @CGUID+77, 4), -- Shattered Hand Warhound
+(@SGGUID+27, @CGUID+2, 0), -- Bonechewer Hungerer
+(@SGGUID+27, @CGUID+3, 1), -- Bonechewer Hungerer
+(@SGGUID+28, @CGUID+33, 0), -- Bleeding Hollow Darkcaster
+(@SGGUID+28, @CGUID+34, 1), -- Bleeding Hollow Darkcaster
+(@SGGUID+29, @CGUID+4, 0), -- Bonechewer Hungerer
+(@SGGUID+29, @CGUID+5, 1), -- Bonechewer Hungerer
+(@SGGUID+30, @CGUID+99, 0), -- Hellfire Sentry
+(@SGGUID+30, @CGUID+100, 1), -- Hellfire Sentry
+
+INSERT INTO `spawn_group_formation`(`SpawnGroupID`, `FormationType`, `FormationSpread`, `FormationOptions`, `MovementID`, `MovementType`, `Comment`) VALUES 
+(@SGGUID+4, 2, '2.2', 0, @CGUID+20, 2, 'Bonechewer Ravener'),
```

Comment:
got some errors when using InstallFullDB.sh again, but no errors when just importing it directly to the database.

Reply:
yea because spawn_group_formation is not part of the instance clear file yet

Reply:
Nah it was much easier, forgot a ; in line 355 :<